### PR TITLE
File Added : Javascript Assignment1

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/JAVASCRIPT/Assignment1/swap.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    var a,b, c;
+    a = Number(document.getElementById('first').value);
+    b = Number(document.getElementById('second').value);
+    c = a;
+    a = b;
+    b = c;
+    document.getElementById('answer1').value = a;
+    document.getElementById('answer2').value = b;
+    }
+</script>
```
Comment (shubhamdalal49) [inline urmila/JAVASCRIPT/Assignment1/swap.html]: put this script tag inside the head tag or before the end of body tag,

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            <div
+                class="form-group col-lg-8"
+                style=" border: solid green; margin-top: 100px; margin-left: 150px; margin-right: 150px; padding: 20px; background-color: rgba(14, 14, 14, 0.5); background-image: url(11.gif);background-repeat: no-repeat; height: 500px; background-size: cover;">
+                <center>
+                    <h2 class="text-warning bg-dark-50">Celsius To Fahrenheit Convert</h2>
+                    <hr />
+                </center>
+                <label for="inputEmail4" class="text-white p-2 bg-dark">Enter the current temperature in Celsius</label>
+                <input type="number" class="form-control col-xl-8" id="num1" placeholder="Enter the current temperature in Celsius" /><br />
+                <!-- <button  class="btn btn-pill btn-success  " onclick="show()">Swap using temporary variable</button>   -->
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment1/temprature.html:29]: remove comment

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="currentColor" class="bi bi-arrow-repeat" viewBox="0 0 16 16">
+                        <path
+                            d="M11.534 7h3.932a.25.25 0 0 1 .192.41l-1.966 2.36a.25.25 0 0 1-.384 0l-1.966-2.36a.25.25 0 0 1 .192-.41zm-11 2h3.932a.25.25 0 0 0 .192-.41L2.692 6.23a.25.25 0 0 0-.384 0L.342 8.59A.25.25 0 0 0 .534 9z"
+                        />
+                        <path
+                            fill-rule="evenodd"
+                            d="M8 3c-1.552 0-2.94.707-3.857 1.818a.5.5 0 1 1-.771-.636A6.002 6.002 0 0 1 13.917 7H12.9A5.002 5.002 0 0 0 8 3zM3.1 9a5.002 5.002 0 0 0 8.757 2.182.5.5 0 1 1 .771.636A6.002 6.002 0 0 1 2.083 9H3.1z"
+                        />
+                    </svg></button
+                ><br />
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment1/temprature.html:35]: check indentation

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                    <h5 id="pid2" class="text-warning bg-dark"></h5>
+                    <br />
+                </div>
+            </div>
+        </div>
+        <script
+            src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+            integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+            crossorigin="anonymous"
+        ></script>
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment1/temprature.html]: script tag proper close

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                document.getElementById("pid2").innerHTML = " Corresponding (" +a+  ") temperature in fahrenheit " + b;
+            }
+        </script>
+    </head>
+    <body style="background-image: url(fire.webp); background-attachment: fixed; background-repeat: no-repeat; background-size: cover">
+        <div class="form-row">
+            <div
+                class="form-group col-lg-8"
+                style=" border: solid green; margin-top: 100px; margin-left: 150px; margin-right: 150px; padding: 20px; background-color: rgba(14, 14, 14, 0.5); background-image: url(11.gif);background-repeat: no-repeat; height: 500px; background-size: cover;">
+                <center>
```
Comment (nileshp1122) [inline urmila/JAVASCRIPT/Assignment1/temprature.html:23]: center tag is outdated

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by vijaya1811 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: urmila/JAVASCRIPT/Assignment1/swap.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            <label>value of b :</label>
+            <input id="second" /><br /><br /><br />
+            <button onclick="swap()">swap</button><br /><br />
+            <label>value of a :</label>
+            <input id="answer1" /> <br /><br />
+            <label>value of b :</label>
+            <input id="answer2" /><br /><br /><br />
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment1/swap.html:77]: where is second method

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        <div class="form-row">
+            <div class="form-group col-lg-8" style="border: solid green; margin-top: 100px; margin-left: 150px; margin-right: 150px; padding: 20px; background-color: rgba(14, 14, 14, 0.5); background-image: url(11.gif); background-repeat: no-repeat; height: 500px; background-size: cover">
+                <center>
+                    <h2 class="text-warning bg-dark-50">Celsius To Fahrenheit Convert</h2>
+                    <hr />
+                </center>
+                <label for="inputEmail4" class="text-white p-2 bg-dark">Enter the current temperature in Celsius</label>
+                <input type="number" class="form-control col-xl-8" id="num1" placeholder="Enter the current temperature in Celsius" /><br />
+                <button class="btn btn-pill btn-success" onclick="show()">Swap using temporary variable</button>
+                <button class="btn btn-success" onclick="show()">
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment1/temprature.html:30]: why you need to buttons

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: urmila/JAVASCRIPT/Assignment1/temprature.html [hunk 8]
```diff
@@ -4,45 +4,33 @@
...
-                            d="M11.534 7h3.932a.25.25 0 0 1 .192.41l-1.966 2.36a.25.25 0 0 1-.384 0l-1.966-2.36a.25.25 0 0 1 .192-.41zm-11 2h3.932a.25.25 0 0 0 .192-.41L2.692 6.23a.25.25 0 0 0-.384 0L.342 8.59A.25.25 0 0 0 .534 9z"
-                        />
-                        <path
-                            fill-rule="evenodd"
-                            d="M8 3c-1.552 0-2.94.707-3.857 1.818a.5.5 0 1 1-.771-.636A6.002 6.002 0 0 1 13.917 7H12.9A5.002 5.002 0 0 0 8 3zM3.1 9a5.002 5.002 0 0 0 8.757 2.182.5.5 0 1 1 .771.636A6.002 6.002 0 0 1 2.083 9H3.1z"
-                        />
+                        <path d="M11.534 7h3.932a.25.25 0 0 1 .192.41l-1.966 2.36a.25.25 0 0 1-.384 0l-1.966-2.36a.25.25 0 0 1 .192-.41zm-11 2h3.932a.25.25 0 0 0 .192-.41L2.692 6.23a.25.25 0 0 0-.384 0L.342 8.59A.25.25 0 0 0 .534 9z" />
+                        <path fill-rule="evenodd" d="M8 3c-1.552 0-2.94.707-3.857 1.818a.5.5 0 1 1-.771-.636A6.002 6.002 0 0 1 13.917 7H12.9A5.002 5.002 0 0 0 8 3zM3.1 9a5.002 5.002 0 0 0 8.757 2.182.5.5 0 1 1 .771.636A6.002 6.002 0 0 1 2.083 9H3.1z" />
                     </svg></button
                 ><br />
```
Comment (Pawan1005) [inline urmila/JAVASCRIPT/Assignment1/temprature.html:35]: create proper indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — COMMENTED** [review]
same code like jestin

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**